ui/redux/selectors/claims.js

```diff
@@ -741,6 +741,22 @@ export const selectPreorderTagForUri = createCachedSelector(selectMetadataForUri
   if (matchingTag) return matchingTag.slice(9);
 })((state, uri) => String(uri));
 
+export const selectRentalTagForUri = createCachedSelector(selectMetadataForUri, (metadata: ?GenericMetadata) => {
+  const matchingTag = metadata && metadata.tags && metadata.tags.find((tag) => tag.includes('rental:'));
+  if (matchingTag) {
+    const trimmedTag = matchingTag.slice(7);
+
+    const tags = trimmedTag.split(':');
```

Comment:
:small_orange_diamond: A bit contrived, but a tag like `ditry_rental:)` would cause issues.
`startsWith` is a better choice than `includes` when slicing is involved.

<img width="387" alt="image" src="https://user-images.githubusercontent.com/64950861/185142977-3d3c044d-afd8-42ab-9154-a23012fc7c24.png">